### DISCUSSION TITLE
Ambigous captions option

### DISCUSSION THREAD

**Discussion by kospl** [vimeo/player.js#693] [Ideas]
### Expected Behavior
To be able to set player option to enable default texttrack (captions) for a video.

### Actual Behavior
Captions cannot be enabled with simple `true/false` option, required to specify language of a `texttrack`, see [embed options](https://github.com/vimeo/player.js/#embed-options).

This causes a lot of pain to enable captions for user:

1. can't easily set one option, like `cc: true` for user who wants captions displayed everywhere.
2. specifying one language (like `en`) in player may result in no captions displayed, if user who uploaded video specified `en-US` or `en-GB` language for their captions.

Then we need to guess which captions we need to show, and it becomes huge amount of code, which could be easily handled by player code if such option like `cc: true` existed. 

Thanks for looking at this feature request.

**Comment by luwes** [comment]
@kospl thanks for the feedback!

Have you looked at https://github.com/vimeo/player.js#gettexttracks-promiseobject-error ?
With the disable and enable api that could easily be used as a on / off toggle I think.

**Reply by kospl** [reply]
Yes sure, I mean writing 10 lines of code and business logic of selecting captions to display is why this feature request exists, it would be much simpler with `bool` setting for captions.

**Comment by kospl** [comment]
This is what we currently have to do to enable captions:

```
var vimeoCaptionsLanguagePriorities = ['en-US', 'en'];

// get all texttracks, then choose which one to use
this.video.player.getTextTracks().then(function(tracks){
	var trackLanguageToUse = null;

	vimeoTexttracksLoop:
	for (var i = 0; i < tracks.length; i++)
	{
		var track = tracks[i];

		for (var j = 0; j < vimeoCaptionsLanguagePriorities.length; j++)
		{
			var langCode = vimeoCaptionsLanguagePriorities[j];

			// looks like this captions we need
			if (track.language === langCode)
			{
				trackLanguageToUse = langCode;
				break vimeoTexttracksLoop;
			}
		}
	}

	if (trackLanguageToUse !== null)
	{
		this.video.player.enableTextTrack(trackLanguageToUse);
	}
	else
	{
		console.error('No captions found for needed language');
	}
}).catch(function(error){
	// an error occurred
	console.error(error);
});
```

looks pretty easy, but with `cc: true` would be a _little_ bit easier.
